### ISSUE TITLE
Unable to sustain a connection between client and Twitch chat

### ISSUE THREAD

**Issue by RobertHawkes** [TwitchLib/TwitchLib#382]
I don't think this is an actual issue with the repository but probably me being a complete newbie to using it.

What I'm trying to do is get the bot to connect however when the bot is running and printing to the screen, it shows that is is connecting but also shows "unaccounted for" messages and then a "Disconnecting Twitch chat client" message. From there, it doesn't show any messages in the twitch channel. My current code is attached, please ignore the references to "TagBot" as that is a separate bot for Discord. My tokens and other sensitive information is stored outside of the program in a JSON file that is read by the information.cs class.

Program.cs: https://pastebin.com/5U91zPnf
TwitchBot.cs: https://pastebin.com/X0fxD7VC
Information.cs: https://pastebin.com/FBnU5GYu

Any help will be much appreciated (I bet you I'm being super dumb and look silly submitting this issue)

**Comment by swiftyspiffy** [comment]
Good catch on this, I didn't even think about that. Thanks for the update!
